diff -urNad highlight-3.4-orig/src/gui-qt/highlight.pro highlight-3.4/src/gui-qt/highlight.pro
--- highlight-3.4-orig/src/gui-qt/highlight.pro	2011-02-09 23:56:23.000000000 +0200
+++ highlight-3.4/src/gui-qt/highlight.pro	2011-04-03 21:34:58.000000000 +0300
@@ -3,8 +3,7 @@
 # -------------------------------------------------
 TARGET = highlight-gui
 TEMPLATE = app
-INCLUDEPATH += . \
-    ../include
+INCLUDEPATH += ../include /usr/include/lua5.1
 win32:CONFIG += static
 CONFIG += precompile_header
 PRECOMPILED_HEADER = precomp.h
