diff -urNad highlight-3.7-orig/src/gui-qt/highlight.pro highlight-3.7/src/gui-qt/highlight.pro
--- highlight-3.7-orig/src/gui-qt/highlight.pro	2011-09-01 22:37:35.000000000 +0300
+++ highlight-3.7/src/gui-qt/highlight.pro	2012-01-06 16:52:19.000000000 +0200
@@ -3,8 +3,7 @@
 # -------------------------------------------------
 TARGET = highlight-gui
 TEMPLATE = app
-INCLUDEPATH += . \
-    ../include
+INCLUDEPATH += ../include /usr/include/lua5.1
 win32:CONFIG += static
 CONFIG += precompile_header
 PRECOMPILED_HEADER = precomp.h
