diff -urNad highlight-3.5-orig/src/gui-qt/highlight.pro highlight-3.5/src/gui-qt/highlight.pro
--- highlight-3.5-orig/src/gui-qt/highlight.pro	2011-06-03 22:31:18.000000000 +0300
+++ highlight-3.5/src/gui-qt/highlight.pro	2011-06-05 18:09:25.000000000 +0300
@@ -3,8 +3,7 @@
 # -------------------------------------------------
 TARGET = highlight-gui
 TEMPLATE = app
-INCLUDEPATH += . \
-    ../include
+INCLUDEPATH += ../include /usr/include/lua5.1
 win32:CONFIG += static
 CONFIG += precompile_header
 PRECOMPILED_HEADER = precomp.h
